libs/application/templates/p-sign/src/forms/application.ts

```diff
@@ -0,0 +1,378 @@
+import {
+  buildForm,
+  buildSection,
+  buildTextField,
+  Form,
+  FormModes,
+  buildExternalDataProvider,
+  buildDataProviderItem,
+  buildMultiField,
+  buildDateField,
+  Application,
+  buildCustomField,
+  buildRadioField,
+  buildDescriptionField,
+  buildFileUploadField,
+  FormValue,
+  buildSelectField,
+  buildDividerField,
+  buildKeyValueField,
+  buildSubmitField,
+} from '@island.is/application/core'
+import { User } from '@island.is/api/domains/national-registry'
+import { format } from 'kennitala'
+import { syslumenn } from './mocks'
+import { QualityPhotoData } from '../types'
+import { NationalRegistryUser, UserProfile } from '../types/schema'
+
+export const getApplication = (): Form => {
+  return buildForm({
+    id: 'PMarkApplicationDraftForm',
+    title: '',
+    mode: FormModes.APPLYING,
+    renderLastScreenButton: true,
+    renderLastScreenBackButton: true,
+    children: [
+      buildSection({
+        id: 'conditions',
+        title: 'Gagnaöflun',
+        children: [
+          buildExternalDataProvider({
+            id: 'approveExternalData',
+            title: 'Gagnaöflun',
+            subTitle: 'Eftirfarandi gögn verða sótt rafrænt með þínu samþykki',
+            description:
+              'Til þess að athuga hvort þú sért með gilt P-merki þá verða upplýsingar verða sóttar  úr kerfi sýslumanna.',
+            checkboxLabel: 'Ég skil að persónuuplýsinga verði aflað ',
+            dataProviders: [
+              buildDataProviderItem({
+                id: 'nationalRegistry',
+                type: 'NationalRegistryProvider',
+                title: 'Læknisvottorð',
+                subTitle: 'Nisi at fermentum adipiscing montes, mi.',
```

Comment:
Is Lorem Ipsum supposed to stay?

Reply:
this is still a draft PR so no need to review this right away :)

Reply:
Although, yes, i might need to use some of the temp texts while this is a WIP